plasmapy/utils/decorators/helpers.py

```diff
@@ -1,12 +1,105 @@
 """
 Miscellaneous decorators for various package uses.
 """
-__all__ = ["preserve_signature"]
+__all__ = ["modify_docstring", "preserve_signature"]
 
 import functools
 import inspect
 
 
+def modify_docstring(func=None, prepend: str = None, append: str = None):
+    """
+    A decorator which programmatically prepends and/or appends the docstring
+    of the decorated method/function.  The unmodified/original docstring is
+    saved as the ``__original_doc__`` attribute.
+
+    Parameters
+    ----------
+    func: callable
+        The method/function to be decorated.
+
+    prepend: `str`
+        The string to be prepended to the ``func``'s docstring.
+
+    append: `str`
+        The string to be appended to the ``func``'s docstring.
```

Comment:
```suggestion
        The string to be prepended to ``func``'s docstring.

    append: `str`
        The string to be appended to ``func``'s docstring.
```
Slight wording change

Reply:
Ha!  This is a Southern California thing.  We are know for putting "the" in front instances like this.  It's believed to come from the fact we've named all of our interstates.  For example, 405 is known as "the San Diego freeway" so we say "the 405."

Since this is so ingrained in me, I can't bring myself to change it.  If it really bothers you, then you'll have sneak the change in yourself. 🤣